lib/vpr/configuration.rb

```diff
@@ -0,0 +1,17 @@
+require 'singleton'
+
+module Vpr
+  class Configuration
+    include Singleton
+
+    attr_accessor :remote
```

Comment:
could you make this accessor private please?

Reply:
Sure!